packages/core/src/animation/Animator.ts

```diff
@@ -0,0 +1,464 @@
+import { AnimatorState } from "./AnimatorState";
+import { InterpolableValueType } from "./enums/InterpolableValueType";
+import { AnimatorRecorderMode } from "./enums/AnimatorRecorderMode";
+import { InterpolableValue } from "./KeyFrame";
+import { AnimatorControllerLayer } from "./AnimatorControllerLayer";
+import { AnimatorController, AnimatorControllerParameter } from "./AnimatorController";
+import { Quaternion, Vector2, Vector3, Vector4 } from "@oasis-engine/math";
+import { Component } from "../Component";
+import { Entity } from "../Entity";
+import { AnimationClip } from "./AnimationClip";
+import { AnimatorUtils } from "./AnimatorUtils";
+import { AnimateProperty } from "./enums/AnimateProperty";
+import { AnimatorLayerBlendingMode } from "./enums/AnimatorLayerBlendingMode";
+import { PlayType } from "./enums/PlayType";
+
+export class Animator extends Component {
+  /**
+   * The mode of the Animator recorder.
+   */
+  recorderMode: AnimatorRecorderMode = AnimatorRecorderMode.Offline;
+  /**
+   * The playback speed of the Animator. 1 is normal playback speed.
+   */
+  speed: number = 1;
+  /**
+   * TODO: The AnimatorControllerParameter list used by the animator.
+   */
+  parameters: AnimatorControllerParameter[] = [];
```

Comment:
If it is not implemented yet, it needs to be deleted.

Reply:
done.